app/components/system_admin/schools/details/view.rb

```diff
@@ -12,7 +12,7 @@ def initialize(school_entity)
 
         def lead_partner?
           if school_entity.is_a?(School)
-            school_entity.lead_school
+            school_entity.lead_partner?
           else
             school_entity.lead_partner
           end
```

Comment:
Is it just me, or does this whole method look a bit strange now? 🤔 Could we neaten this up at all?

Reply:
Good shout, I added an alias to the school form called lead_partner?, so now the method name is the same in both cases. So now we can get rid of this completely and replace it with `delegate :lead_partner?, to: :school_entity`. See the commit [here](https://github.com/DFE-Digital/register-trainee-teachers/pull/4577/commits/636061bfc23c2cafc858783f60214e0fea3f07f2).